libraries/chain/webassembly/crypto.cpp

```diff
@@ -246,4 +249,88 @@ namespace eosio { namespace chain { namespace webassembly {
       return return_code::success;
    }
 
+   bool interface::bls_verify( span<const char> signature, span<const char> digest, span<const char> pub) const {
+
+
+      fc::crypto::blslib::bls_signature u_sig;
+      fc::crypto::blslib::bls_public_key u_pub;
+      vector<uint8_t> u_digest;
+
+      datastream<const char*> s_sig( signature.data(), signature.size() );
+      datastream<const char*> s_pub( pub.data(), pub.size() );
+      datastream<const char*> s_digest( digest.data(), digest.size() );
+
+      fc::raw::unpack( s_sig, u_sig );
+      fc::raw::unpack( s_pub, u_pub );
+      fc::raw::unpack( s_digest, u_digest );
+
+      std::cerr << u_pub.to_string() << "\n";
+      std::cerr << u_sig.to_string() << "\n";
+
+      bool result = fc::crypto::blslib::bls_utils::verify(u_pub, u_digest, u_sig);
+
+      return result;
+
+   }
+
+   int32_t interface::bls_aggregate_pubkeys( span<const char> pubkeys, span<char> aggregate) const {
```

Comment:
Similar comment for all these aggregate host functions; unless we can handle ~1 million inputs (33MiB worth of inputs) in 1-2 milliseconds, we need to checktime (or add a subjective limit).